### PR TITLE
(role/ccs-mcm) install tomcat library jar files

### PR DESCRIPTION
**Pull request by glennmorris**

### CODE REVIEW

File: site/profile/manifests/ccs/tomcat.pp [hunk 1]
```diff
@@ -80,6 +85,19 @@
     }
   }
 
+  unless (empty($jars)) {
```
Comment (jhoblitt) [inline site/profile/manifests/ccs/tomcat.pp]: The unless isn't needed here as `{}.each` is a noop.

**Review by jhoblitt — CHANGES_REQUESTED** [review]

File: site/profile/manifests/ccs/tomcat.pp (hunk 1 above)
Comment (glennmorris) [reply site/profile/manifests/ccs/tomcat.pp]: That's what I get for reusing the [existing style](https://github.com/lsst-it/lsst-control/blob/production/site/profile/manifests/ccs/tomcat.pp#L74)...

I have pushed a revised version.

Comment (jhoblitt) [reply site/profile/manifests/ccs/tomcat.pp]: Ya. You can't trust that guy.

**Review by jhoblitt — APPROVED** [review]
LGTM.

**Review by tony-johnson — COMMENTED** [review]
While we are thinking about this, lets update to the latest version of CCSWebTrending. I will look up what that is.

**Comment by glennmorris** [comment]
> While we are thinking about this, lets update to the latest version of CCSWebTrending. I will look up what that is.

We are already using the latest release in nexus (2.1).

**Comment by tony-johnson** [comment]
> > While we are thinking about this, lets update to the latest version of CCSWebTrending. I will look up what that is.
> 
> We are already using the latest release in nexus (2.1).

Yes, but that is because I have been cheating and pushing out changes without using puppet or the release process. I have just created a 2.2 release.

**Review by tony-johnson — APPROVED** [review]
Looks good, thanks

**Pull request merged**